tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -609,7 +610,30 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       result.getErrors.getGroups should have length 2
       result.getErrors.getGroups.head.getId.getId shouldBe "badgroupid_frompolicy"
       result.getErrors.getGroups.tail.head.getId.getId shouldBe s"data_recovery_${primaryTestUserId.getId}"
+    }
 
+    "clear an empty policy" in {
+      val data: Array[Byte] = List(1, 2, 3).map(_.toByte).toArray
+      val id = GroupId.validate(s"data_recovery_${primaryTestUserId.getId}")
```

Comment:
A comment as to why this id is important might be nice for the future.

Reply:
Added